static/.htaccess

```diff
@@ -14,7 +14,8 @@ RewriteRule ^docs(/(8.0|next))?/components/modeler/desktop-modeler/element-templ
 # Added Self-Managed Modeler redirect
 RewriteRule ^docs/next/self-managed/web-modeler/installation/(.*)$ /docs/next/self-managed/modeler/web-modeler/installation/$1 [R=301,L]
 
-RewriteRule ^docs/next/apis-clients/working-with-apis-clients/?$ /docs/next/apis-clients/working-with-apis-tools/ [R=301,L]
+# this looks inverted, but it's because we had a redirect from clients -> tools, and we're changing it back from tools -> clients
+RewriteRule ^docs/next/apis-clients/working-with-apis-tools/?$ /docs/next/apis-clients/working-with-apis-clients/ [R=301,L]
```

Comment:
This is probably the weirdest part of the PR -- we pushed out a redirect already to send them from apis-clients -> apis-tools, so if they saved that URL we want to send them _back_ to apis-clients. I think this could result in a redirect loop for some users -- if their browser had cached the apis-clients -> apis-tools redirect, and now they get an apis-tools -> apis-clients redirect. 

I don't know a way around this. I think it will affect very very few people. Clearing browser cache would resolve it. I'm not sure if that's something we'll need to communicate somehow, or where to communicate it.

Reply:
C8 Slack and Forum post for externals. ask-documentation for internal. Keep ears open 🤷‍♀️ 

As much as we try to avoid significant disruptions, we may have them.

Reply:
The more I think about this, the more I think the correct move is to move all the old versions to apis-tools. 

If we revert now, we'll end up re-reverting in the near future, to point everything _back_ at apis-clients again. And then we'd have to warn people about redirect loops _again_. 

Or, we put everything in the end-state of apis-tools, and we wash our hands of this. It's a tiny bit more work (I think) than what I already did in this PR, but I think it's the smarter move long-term. 

I'm going to work on that now -- if it turns into a rabbit hole, I will bail quickly, and merge this PR as-is instead.

Reply:
Ok, I just pushed up the inverted scenario. Now everything is changed to `/apis-tools/working-with-apis-tools`.

Reply:
This looks like it's working when I test locally. LMK how else I can support.